api-spec/protobuf/operator.proto

```diff
@@ -362,3 +369,34 @@ message WebhookInfo {
   string endpoint = 2;
   bool is_secured = 3;
 }
+
+
+message ListDepositsRequest{
+  int64 account_index = 1;
+  Page page = 2;
+}
+message ListDepositsReply{
+  int64 account_index = 1;
+  repeated TxOutpoint deposits = 2;
```

Comment:
would be better to be:
```suggestion
  repeated UtxoInfo deposits = 2;
```

since it that kind of message gives also info about the asset and amount deposited.